Userland/Applications/Browser/InspectorWidget.h

```diff
@@ -67,6 +69,7 @@ class InspectorWidget final : public GUI::Widget {
     Web::Layout::BoxModelMetrics m_node_box_sizing;
 
     Optional<DeprecatedString> m_dom_json;
+    Optional<DeprecatedString> m_accessibility_json;
```

Comment:
No need to store the JSON string, checking the string for changes before updating the model seems like a premature optimisation (since this is not frequently updated anyway).